src/components/@shared/atoms/Loader/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import Loader from '@shared/atoms/Loader'
+
+export default {
+  title: 'Component/@shared/atoms/Loader',
+  component: Loader
+} as ComponentMeta<typeof Loader>
+
+const Template: ComponentStory<typeof Loader> = (args) => <Loader {...args} />
+
+interface Props {
+  args: {
+    message: string
+  }
+}
+
+export const Primary: Props = Template.bind({})
+Primary.args = {
```

Comment:
please not use hierarchical terms for something which has no hierarchy. We do not have a primary or secondary loader. We always have a default state of a component which is a component with only the required props passed. Then we call that `Default`. Every additional story then should get meaningful names, so in this case we could have `Default` and `WithMessage`